test/deck_test.exs

```diff
@@ -2,11 +2,60 @@ defmodule DeckTest do
   use ExUnit.Case
   doctest Deck
 
-  @ordered_deck ["2c", "2d", "2h", "2s", "3c", "3d", "3h", "3s", "4c", "4d", "4h", "4s",
-                 "5c", "5d", "5h", "5s", "6c", "6d", "6h", "6s", "7c", "7d", "7h", "7s",
-                 "8c", "8d", "8h", "8s", "9c", "9d", "9h", "9s", "Tc", "Td", "Th", "Ts",
-                 "Jc", "Jd", "Jh", "Js", "Qc", "Qd", "Qh", "Qs", "Kc", "Kd", "Kh", "Ks",
-                 "Ac", "Ad", "Ah", "As"]
+  @ordered_deck [
+    "2c",
```

Comment:
I understand that this is done with auto-format but can you please change it back?

I would also like to see some tests for this new method.